app/controllers/photos_controller.rb

```diff
@@ -8,6 +8,7 @@ def find_album
 
   def find_photo
     @photo = Photo.find(params[:id])
+    @photo.view += 1;
```

Comment:
I think that including this method here will increment the view count by one every time you call a new, create or index method in the photos controller (so a newly added photo will already have 2 views). So I think you can delete this line, and just leave the method on line 22.